english/yoga/lifestyle/yoga_lifestyle.md

```diff
@@ -12,4 +12,7 @@ they are in constant change.
 
 They describe a temporary occurrence. Our essence remains the same. 
 Our core values determine who we are and if we are living a yoga 
-lifestyle.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.